tensorboard/webapp/metrics/views/main_view/common_selectors_test.ts

```diff
@@ -65,6 +79,114 @@ describe('common selectors', () => {
         metrics: null,
       },
     };
+
+    runTableItems = [
+      {
+        run: {
+          id: 'run1-id',
+          name: 'run1',
+          startTime: 0,
+          hparams: [
+            {
+              name: 'accurracy',
+              value: 1,
+            },
+          ],
+          metrics: null,
+        },
+        experimentAlias: {
+          aliasNumber: 1,
+          aliasText: 'exp1',
+        },
+        experimentName: 'experiment1',
+        selected: true,
+        hparams: new Map<string, number>([['lr', 5]]),
+        metrics: new Map<string, number>([['foo', 1]]),
+      },
+      {
+        run: {
+          id: 'run2-id',
+          name: 'run2',
+          startTime: 0,
+          hparams: [
+            {
+              name: 'accurracy',
+              value: 1,
+            },
+          ],
+          metrics: null,
+        },
+        experimentAlias: {
+          aliasNumber: 1,
+          aliasText: 'exp1',
+        },
+        experimentName: 'experiment1',
+        selected: true,
+        hparams: new Map<string, number>([['lr', 3]]),
+        metrics: new Map<string, number>([['foo', 2]]),
+      },
+      {
+        run: {
+          id: 'run1-id',
```

Comment:
Shouldn't this be run3?

Reply:
I wanted to have a run with the same id associated with two different experiments

Reply:
I thought run Ids were universally unique. We even have a selector which gets an experimentId from a runId.